### PR TITLE
feat: chainsec fixes

### PR DESCRIPTION
**Pull request by Schlagonia**

### CODE REVIEW

File: ape-config.yaml [hunk 1]
```diff
@@ -21,20 +21,27 @@ dependencies:
   - name: yearn-vaults
     github: yearn/yearn-vaults-v3
     ref: v3.0.1
+    config_override:
```
Comment (fp-crypto) [inline ape-config.yaml:24]: DELETE

File: contracts/debtAllocators/DebtAllocator.sol [hunk 2]
```diff
@@ -293,7 +304,7 @@ contract DebtAllocator {
 
             // Check if it's over the threshold.
             if (toPull > minimumChange) {
-                // Can't lower debt if there is unrealised losses.
```
Comment (fp-crypto) [inline contracts/debtAllocators/DebtAllocator.sol:296]: Should everything be singular?

File: contracts/debtAllocators/DebtAllocator.sol [hunk 3]
```diff
@@ -410,7 +421,7 @@ contract DebtAllocator {
 
     /**
      * @notice Set the minimum change variable for a strategy.
-     * @dev This is the amount of debt that will needed to be
+     * @dev This is the minimum amount of debt that will have to be
```
Comment (fp-crypto) [inline contracts/debtAllocators/DebtAllocator.sol:424]:
```suggestion
     * @dev This is the minimum amount of debt to be
```

**Review by fp-crypto — APPROVED** [review]
Approved with comments

File: contracts/debtAllocators/DebtAllocator.sol (hunk 2 above)
Comment (Schlagonia) [reply contracts/debtAllocators/DebtAllocator.sol:296]: probably but its too late

**Pull request merged**